cmd/plume/fedora.go

```diff
@@ -80,7 +80,7 @@ var (
 			Boards:  awsFedoraBoards,
 			AWS: awsSpec{
 				BaseName:        "Fedora",
-				BaseDescription: "Fedora {{.ImageType}} AMI",
+				BaseDescription: "Fedora Cloud Base AMI",
```

Comment:
Does it make sense to drop the `ImageType` parameter and always hard code in the `Image` field as well?

Reply:
I think it may, but I wasn't sure so I didn't touch it yet

Reply:
That's fine, we can always come back in a later PR and drop it if it's unnecessary.

Reply:
cool. Thanks